### DISCUSSION TITLE
Use 32c64t (2.5GHz) and 36c72t (3.2GHz) the same speed

### DISCUSSION THREAD

**Discussion by guitengyue** [Chia-Network/bladebit#138] [Q&A]
Question one：
e2683v4 x2(32c64t,2500Mhz), 16x32g ddr4 2133, total 11-12min
e2696v3 x2(36c72t,3200Mhz(I use the e5 bug)),16x32g ddr4 2133, also total 11-12min

Question two：
And I find when use 16x32g,the memry read and write have more then 100GB/s, when I use 14x32g,the memery read and write only have 50GB/s, bug the total time is the same 11-12min.....

**Comment by harold-b** [comment]
Moving this to discussions.

It would be too hard to tell without any extra info, and testing. But you might try a straight memory read/write test between both system to see if you see the same discrepancy.

**Comment by mmitech** [comment]
Answer to question 2: your CPUs support and work best in quad-channel mode, if you use 14x32 they don't work on quad mode and thus you lose the full bandwidth.

Question1: I also have a similar issue, I have 2 machines, one with fewer and the other with more cores, but they pretty much have the same performance (similar clock), I have no idea why! I am trying to figure out the trick as well.

**Reply by guitengyue** [reply]
I know the reason why the memory read and write speed is 100G and 50G..
Question 2 I mean, why is the speed of the plot the same in 4-channel and non-4-channel, they all takes 11-12min....According to my test, I think that as long as the memory speed can reach 50G, it is already the upper limit.

**Reply by guitengyue** [reply]
Can you provide the specific parameters of the CPU of the two devices?

**Reply by mmitech** [reply]
I also don't know what is the reason behind that, I have 2 setups, one with quad Xeon E5-4640 (2.4Ghz 8 core) and the other has quad Xeon E5-4657L V2 (2.4GHz 12 core), they Both do ~ 16 minutes/Plot
